app/javascript/components/close_contact/CloseContact.js

```diff
@@ -120,8 +120,10 @@ class CloseContact extends React.Component {
                   className="form-square"
                   value={this.state.notes || ''}
                   placeholder={this.closeContactNotePlaceholder}
+                  maxLength="2000"
```

Comment:
Any reason not to put this same limit (and count) on the enrollment text field now? As it is right now, the user has no way of knowing how far over they are when they see the `yup` validation error.

Reply:
Nope. I can add it there as well.

Reply:
Added this in a new commit.